code/controllers/subsystem/timer.dm

```diff
@@ -212,6 +212,7 @@ SUBSYSTEM_DEF(timer)
 		if(!(qtimer.flags & TIMER_LOOP))
 			qdel(qtimer)
 		else
+			bucket_count++
```

Comment:
how does this fix it

Reply:
The code a few lines above have already removed this timer from `bucket_count` but `bucketEject` when `spent` is 0 also removes it. It needs to be 0 for other reasons so we just add to `bucket_count` like `QDELETED` does